src/main/java/org/isf/users/rest/UserController.java

```diff
@@ -0,0 +1,285 @@
+/*
+ * Open Hospital (www.open-hospital.org)
+ * Copyright © 2006-2023 Informatici Senza Frontiere (info@informaticisenzafrontiere.org)
+ *
+ * Open Hospital is a free and open source software for healthcare data management.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * https://www.gnu.org/licenses/gpl-3.0-standalone.html
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <https://www.gnu.org/licenses/>.
+ */
+package org.isf.users.rest;
+
+import jakarta.validation.Valid;
+
+import java.util.List;
+import java.util.Objects;
+import java.util.stream.Collectors;
```

Comment:
The import order doesn't match project standards; see https://github.com/informatici/openhospital-api/blob/develop/.ide-settings/eclipse/OpenHospital.importorder

6=
5=com
4=org
3=jakarta
2=javax
1=java